pkgs/applications/version-management/p4v/default.nix

```diff
@@ -15,21 +15,20 @@ stdenv.mkDerivation rec {
   ldLibraryPath = lib.makeLibraryPath [
       stdenv.cc.cc.lib
       qtbase
-      qtmultimedia
       qtscript
-      qtsensors
+      qtwebengine
```

Comment:
Any chance this already works without qtwebkit?

Reply:
Upstream ships `libQt5WebKit.so` and `libQt5WebEngine.so`, so I guess both are needed.